about.md

```diff
@@ -4,18 +4,53 @@ title: About
 is_nav: true
 permalink: /about/
 ---
+開発、改善、海賊
 
+Astral Dynamics is an Innovation and Rapid Proto-typing agency. We are a multi-
+disciplinary team that brings genuine creativity together with exception 
+technical knowledge and skill. We respond to challenges, design solutions and rapid prototype.
 
-Astral Dynamics is a forward thinking tech startup with a focus on improving and enhancing culture.
+We follow our own Agile methodology and have a unique organisational culture based 
+on the naval metaphore. Every project is a voyage of discovery. We believe very 
```

Comment:
There is no "the naval metaphor".  We are working on following _a naval metaphor_, but we can't just use the term without referencing it.